meteor/meteor.d.ts

```diff
@@ -1,435 +1,673 @@
 // Type definitions for Meteor 1.3
 // Project: http://www.meteor.com/
-// Definitions by: Dave Allen <https://github.com/fullflavedave>
+// Definitions by: Alex Borodach <https://github.com/barbatus>,
```

Comment:
`// Definitions by: Dave Allen <https://github.com/fullflavedave>, Alex Borodach <https://github.com/barbatus>`
please use this format.